libarchive/test/test_read_format_tar_empty_with_gnulabel.c

```diff
@@ -45,7 +45,7 @@ DEFINE_TEST(test_read_format_tar_empty_with_gnulabel)
 
 	/* Verify that the format detection worked. */
 	assertEqualInt(archive_filter_code(a, 0), ARCHIVE_FILTER_NONE);
-	assertEqualInt(archive_format(a), ARCHIVE_FORMAT_TAR);
+	assertEqualInt(archive_format(a), ARCHIVE_FORMAT_TAR_GNUTAR);
```

Comment:
The earlier format detection means that we know the precise format at this point now.